.gitignore

```diff
@@ -11,3 +11,6 @@ out/
 # gradle
 .gradle
 build/
+
```

Comment:
Thanks.  I've been a little lazy setting up the appropriate tab expansion
recently.  The main difficulty is that it tends to change across
files/projects, so one file will want tabs expanded to 4 spaces, another to
2 ...

I think I'll just have to be more careful in future / make my whitespace
visible.

On Tue, Jul 21, 2015 at 5:56 PM Adam <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/mesosphere/hdfs/pull/157#discussion_r35171980>:
>
> > @@ -11,3 +11,6 @@ out/
> >  # gradle
> >  .gradle
> >  build/
> > +
>
> I use vim! And for other projects, I also .gitignore *.swp
> You can also set expandtab in your .vimrc to insert spaces instead of
> tabs.
> I can share my .vimrc if it'd help you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/hdfs/pull/157/files#r35171980>.
>